------------------------------------------ 
test_database 待修改项：

ojdbc6 11.2.0.4.0 ：
	Jndi（javax.naming.NameNotFoundException: Name [jndi/oracle] is not bound in this Context. Unable to find [jndi].配置有问题）

db2jcc4 10.1 : 
	Jndi（javax.naming.NameNotFoundException: Name [DB2] is not bound in this Context. Unable to find [DB2].配置有问题）

postgresql 9.3.1104-jdbc4 : 
	Druid(驱动版本有问题42.2.2版本可以正常运行) 
	C3p0(驱动版本有问题42.2.2版本可以正常运行)
	Jdbc Statement（call proc_update('Jdbc')存储过程有异常rg.postgresql.util.PSQLException: ERROR: query has no destination for result data建议：If you want to discard the results of a SELECT, use PERFORM instead.在位置：PL/pgSQL function proc_update(character varying) line 3 at SQL statement）  	
	Jndi（javax.naming.NameNotFoundException: Name [jdbc/postgresql] is not bound in this Context. Unable to find [jdbc].配置有问题）

mysql-connector-java 5.1.39 :  
	Jndi（Name [jdbc/mysql] is not bound in this Context. Unable to find [jdbc].配置有问题）

sqlserver jtds 1.3.1 : 
	Jndi（Name [SqlServerDS] is not bound in this Context. Unable to find [SqlServerDS].配置有问题）

derby 10.9.1.0 : 
	Jdbc Statement（java.sql.SQLNonTransientConnectionException: java.net.ConnectException：连接到端口 1,527 上的服务器 192.168.2.214 时出错，消息为 Connection refused: connect。）
	Jdbc PreparedStatement（java.sql.SQLNonTransientConnectionException: java.net.ConnectException：连接到端口 1,527 上的服务器 192.168.2.214 时出错，消息为 Connection refused: connect。）

dm : 
	Jdbc Statement（java.sql.SQLException: 用户名或密码错误）

-----------------------------------------------------------------